libpod/container.go

```diff
@@ -996,3 +996,13 @@ func (c *Container) IsInfra() bool {
 func (c *Container) IsReadOnly() bool {
 	return c.config.Spec.Root.Readonly
 }
+
+// NetworkDisabled returns whether the container is running with a disabled network
+func (c *Container) NetworkDisabled() bool {
+	for _, ns := range c.config.Spec.Linux.Namespaces {
+		if ns.Type == spec.NetworkNamespace {
+			return ns.Path == ""
```

Comment:
would this return a false positve when `PostConfigureNetNS` is used?

I think `PostConfigureNetNS` must be checked before looking into the namespaces, and return immediately `false` if it is used

Reply:
I can do this, but isn't this just indicating that the system is in rootless mode? What happens if I do 
podman run --net=none 
WHile in rootless mode?